xml/depl_conf_admin_crowbar.xml

```diff
@@ -2038,14 +2053,14 @@ pools, gateway, routing, and virtual switches to your new network.
 Set up interface mapping using either Open vSwitch (OVS) or Linuxbridge.
 For Open vSwitch run the following command:
```

Comment:
this might make more sense as an `<itemizedlist/>`:

* For OvS, do ...
* For Linuxbridge, do ...